spine-c/include/spine/Animation.h

```diff
@@ -60,6 +60,8 @@ typedef struct spAnimation {
 } spAnimation;
 
 spAnimation* spAnimation_create (const char* name, int timelinesCount);
+spAnimation* spAnimation_createWithTimelines (const char* name, float duration,
```

Comment:
We can. The only downside is the allocating of a new array for timelines instead of using existing one. Though sometimes it can be allocated with the current implementation too (if trimming of `kvec` will not be able to do it in place).